### PR TITLE
docs(snippet): import correctly

### PR DESCRIPTION
**Pull request by Haroenv**
**Summary**

imports from the wrong package for custom hits

**Result**

import from connectors, then refactored not to use connectors

### CODE REVIEW

File: packages/react-instantsearch/src/widgets/Snippet.js [hunk 1]
```diff
@@ -15,29 +15,24 @@ import SnippetComponent from '../components/Snippet.js';
...
- *    <InstantSearch
+ *   return (
+ *     <InstantSearch
  *       appId="latency"
  *       apiKey="6be0576ff61c053d5f9a3225e2a90f76"
  *       indexName="ikea"
  *     >
- *       <CustomHits />
+ *       <Hits
+ *         hitComponent={hit => (
```
Comment (samouss) [inline packages/react-instantsearch/src/widgets/Snippet.js]: The parameter hit are the props of the component, so you can't access it directly. You should use destructuring:

```jsx
<Hits
  hitComponent={({ hit }) => (
    ...
  )}
/>
```

File: packages/react-instantsearch/src/widgets/Snippet.js [hunk 2]
```diff
@@ -15,29 +15,24 @@ import SnippetComponent from '../components/Snippet.js';
...
- * <div>
- *   {hits.map(hit =>
- *     <p key={hit.objectID}>
- *       <Snippet attributeName="description" hit={hit} />
- *     </p>
- *   )}
- * </div>
- * );
- *
+ * import { Snippet, InstantSearch, Hits } from "react-instantsearch/dom";
```
Comment (samouss) [inline packages/react-instantsearch/src/widgets/Snippet.js]: Use single quote as the reset of the example.

Comment (Haroenv) [reply packages/react-instantsearch/src/widgets/Snippet.js]: my prettier settings were on default

**Comment by samouss** [comment]
Prettier is still not happy... See the CI log.

**Pull request merged**